### DISCUSSION TITLE
[@types/node] setMaxListeners static method missing from node 16 types

### DISCUSSION THREAD

**Discussion by achingbrain** [DefinitelyTyped/DefinitelyTyped#61006] [Issues with a @types package] [answered]
The static [`setMaxListeners`](https://nodejs.org/api/events.html#eventssetmaxlistenersn-eventtargets) function was added to `node@15.4.0` but it's not in `@types/node@16.x.x`.

It is in `@types/node@18.x.x`, so that's good, but if you depend on that package you may end up accidentally using features that aren't in `node@16.x.x` which causes problems if you want to support `node@16.x.x`.

[a comment by an automated account is not shown]

**Comment by mcollina** [comment, marked as answer]
Would you like to send a Pull Request?

**Reply by achingbrain** [reply]
Here you go: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61007
